content/dpg indicators/3/approaches.en.adoc

```diff
@@ -51,7 +51,7 @@ What makes this approach unique?
 * *Flexibility*:
   Offer additional pathways to customize or integrate the product with other technologies or business needs.
 
-A common example of this is the https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux[Red Hat Enterprise Linux operating system] and its Open Source base, link:++{{< ref "communities/fedora" >}}++[Fedora Linux].
+A common example of this is the https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux[Red Hat Enterprise Linux operating system] and its Open Source base, link:++{{< ref "dpg indicators/8/communities/fedora" >}}++[Fedora Linux].
 
 [[approach--services-around-product]]
 === Services around your product
```

Comment:
As a reminder, all renamed pages will need to include [aliases](https://gohugo.io/content-management/urls/#aliases) to their old paths, so a reader with a saved URL or bookmark to an old path will be redirected to the new path.

Reply:
Oh, at that time i didn't understand it well but i got it now👍

Reply:
@jwflory But, I think most of the previous paths are changed.
Then, I think it requires us to add aliases for a lot.
Is there any other way or do we just have to do it for all of them?

Reply:
@Neha9849 Following up on our call this week, we will need to create aliases manually in each page. It is a necessary step for each page since we are restructuring all content.

Reply:
Addressed in commits 0eabde03d98eda5e3fb0debea1bfa5781f8440e3 and 547525c0998b1c9de04fe0afeda2cf0bc66abf87.